lnrpc/lightning.proto

```diff
@@ -652,6 +664,40 @@ message SendCustomMessageResponse {
     string status = 1;
 }
 
+message SubscribeOnionMessagesRequest {
+}
+
+message OnionMessage {
+    // Peer from which the message originates
+    bytes peer = 1;
+
+    // BlindingPoint is the route blinding ephemeral pubkey to be used for the
+    // onion message.
+    string blinding_point = 2;
+
+    // Onion is the raw serialized mix header used to relay messages in a
```

Comment:
Q: what do you mean by `mix header` I mean this is the body of the message why do you call it header ?

Reply:
"mix header" is the term used in the original Sphinx paper. The "mix header" is the part that contains the per-hop encrypted instructions for each node along the route. I find the usage here confusing as well. (It's copy-pasta from Carla's code, but I'll take responsibility for any confusion :smile:).

I'll update it to this:

```
// Onion is the raw serialized Sphinx onion packet containing the
// per-hop encrypted routing information used to relay this message
// through the network in a privacy-preserving manner.
```